src/screens/NestedInfoScreen.tsx

```diff
@@ -0,0 +1,118 @@
+import { StackScreenProps } from '@react-navigation/stack';
+import React, { useCallback, useContext, useState } from 'react';
+import { ActivityIndicator, RefreshControl, SectionList } from 'react-native';
+
+import {
+  HtmlView,
+  LoadingContainer,
+  LoadingSpinner,
+  SafeAreaViewFlex,
+  SectionHeader,
+  Wrapper,
+  WrapperWithOrientation
+} from '../components';
+import { colors } from '../config';
+import { useStaticContent } from '../hooks';
+import { useRenderItem } from '../hooks/listHooks';
+import { NetworkContext } from '../NetworkProvider';
+import { QUERY_TYPES } from '../queries';
+
+type NestedInfo = {
+  content?: string;
+  title: string;
+  children: Array<{
+    title: string;
+    routeName: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    params?: any;
+  }>;
+};
+
+const ListHeaderComponent = ({ loading, html }: { loading: boolean; html?: string }) => {
+  if (loading) {
+    return <LoadingSpinner loading />;
+  }
+
+  if (!html?.length) {
+    return null;
+  }
+
+  return (
+    <Wrapper>
+      {/* @ts-expect-error HtmlView uses memo in js, which is not inferred correctly */}
+      <HtmlView html={html} />
+    </Wrapper>
+  );
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const NestedInfoScreen = ({ navigation, route }: StackScreenProps<any>) => {
+  const { isConnected } = useContext(NetworkContext);
+  const [refreshing, setRefreshing] = useState(false);
+  const name = route.params?.name;
+  const rootRouteName = route.params?.rootRouteName;
+  const navigationTitle = route.params?.title;
+  const { data, loading, refetch } = useStaticContent<NestedInfo>({ name, type: 'json' });
+  const { data: dataHtml, loading: loadingHtml, refetch: refetchHtml } = useStaticContent<string>({
+    name: data?.content ?? '',
+    type: 'html',
+    skip: !data?.content
+  });
+
+  const onRefresh = useCallback(async () => {
+    setRefreshing(true);
+    if (isConnected) {
+      await refetch?.();
+      await refetchHtml?.();
+    }
+    setRefreshing(false);
+  }, [isConnected, refetch]);
+
+  const renderItem = useRenderItem(QUERY_TYPES.PUBLIC_JSON_FILE, navigation);
+
+  if (loading && !data)
+    return (
+      <LoadingContainer>
+        <ActivityIndicator color={colors.accent} />
+      </LoadingContainer>
+    );
+
+  // aggregate current rootRouteName and title for navigation, if they are not present in params of children
+  const sectionData = [
+    {
+      title: data.title,
+      data: data.children?.map((child) => {
+        return {
+          ...child,
+          params: {
+            ...child.params,
+            rootRouteName: child.params?.rootRouteName ?? rootRouteName,
+            title: child.params?.title ?? navigationTitle
```

Comment:
please check against the navigation title für the screen entering per "loop" from home screen. i have an empty one and could not make it appear. i have added a `title` inside child params, but this just applies for the second time navigated to "loop".

Reply:
the title is set (as with other about links as well) through the about home static content. because of that there is no initial title set.
there is an initial title, when you open it through the drawer (because it is set as part of the drawer content).

does that explain/solve this?

Reply:
yes, thanks, works, knot untied